llvm/test/MC/AsmParser/preserve-comments-aarch64-linux.s

```diff
@@ -0,0 +1,10 @@
+	// REQUIRES: aarch64-registered-target
```

Comment:
excess \t at the line beginning? If this is intentional, add a comment.

If this

Reply:
`diff -b` is not happy otherwise on some distros,
so this is similar to other tests with such formatting.

Reply:
This comment still looks vague... `//` is the universal comment marker, but AArch64 sets `CommentString` to `//`, so it will print `//`.
However, it seems that this MC test passes even before this change.

I don't think `// RUN: llvm-mc -preserve-comments -n -triple aarch64 %s -o %t` (removed unneeded `<` and used `-o` for more conventional tests) needs to be indented